docs/pages/database-access/guides/oracle-self-hosted.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Database Access with Oracle (Preview)
+description: How to configure Teleport Database Access with Oracle.
+---
+
+<Admonition type="warning" title="Preview">
+  Database Access for Oracle Server is currently in Preview mode.
+</Admonition>
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["cloud", "enterprise"]}
+  scopeOnly={true}
+  min="12.3"
+>
+  Database access for Oracle is available starting from Teleport `v12.3`.
+</Details>
+
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up Teleport to access your self-hosted Oracle server 18c or later.
+- Connect to your database through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Redis Self-Hosted](../../../img/database-access/guides/oracle_selfhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Redis Cloud](../../../img/database-access/guides/oracle_selfhosted_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- Self-hosted Oracle server instance 18c or later.
+- The `sqlcl` [Oracle client](https://www.oracle.com/pl/database/sqldeveloper/technologies/sqlcl/) installed and added to your system's `PATH` environment variable or any GUI client that supports JDBC
+  Oracle thin client.
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+<ScopedBlock scope={["enterprise"]}>
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Proxy Service:
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:3080 \
```

Comment:
Is there an equivalent flow with `teleport db configure` + `teleport start` that would work here?

We're trying to get away from `--auth-server=proxy.server.address.here`, as it's pretty confusing to newcomers.